SPECS-EXTENDED/xmldb-api/xmldb-api.spec

```diff
@@ -22,14 +22,13 @@ Distribution:   Mariner
 %define cvs_version 20041010
 Name:           xmldb-api
 Version:        0.1
-Release:        27%{?dist}
+Release:        28%{?dist}
 Summary:        XML:DB API for Java
 License:        Apache-1.1
-Group:          Development/Libraries/Java
```

Comment:
I think you accidentally removed the `Group` tag.

Reply:
I actually removed it on purpose. AFAIK Group tags serve no purpose and are not standardized in any way, so I figured removing that field would clean the SPEC up a bit.